src/Overlay.js

```diff
@@ -120,12 +130,73 @@ function wrapRefs(props, arrowProps) {
     aRef.__wrapped || (aRef.__wrapped = (r) => aRef(findDOMNode(r)));
 }
 
-function Overlay({ children: overlay, transition, ...outerProps }) {
+const marginModifiers = [
+  {
+    name: 'marginOffsets',
+    enabled: true,
+    phase: 'beforeRead',
+    effect: ({ state }) => {
+      return () => {
+        state.elements.reference.removeAttribute('aria-describedby');
+      };
+    },
+    fn: ({ state, name }) => {
+      const { popper: popperRect } = state.rects;
+      const { popper, arrow } = state.elements;
+
+      const popperMargins = getMargins(popper);
+      const arrowMargins = arrow && getMargins(arrow);
+
+      state.rects.popper = {
```

Comment:
this needs to be set very early since all the other modifiers use it for calculations

Reply:
Yup, very early it is. Tested the other phases as well but no go on those.